torch/profiler/profiler.py

```diff
@@ -242,12 +242,12 @@ def export_stacks(self, path: str, metric: str = "self_cpu_time_total"):
     def toggle_collection_dynamic(
         self, enable: bool, activities: Iterable[ProfilerActivity]
     ):
-        """Toggle collection of activities on/off
+        """Toggle collection of activities on/off. Currently supports toggling Torch Ops (CPU) and all CUDA activity
+        supported in Kineto
 
         Args:
             activities (iterable): list of activity groups (CPU, CUDA) to use in profiling, supported values:
-                ``torch.profiler.ProfilerActivity.CPU``, ``torch.profiler.ProfilerActivity.CUDA``,
-                ``torch.profiler.ProfilerActivity.XPU``.
+                ``torch.profiler.ProfilerActivity.CPU``, ``torch.profiler.ProfilerActivity.CUDA``
```

Comment:
why do we remove XPU, could you add a comment if its not supported?

Reply:
XPU and CPU were erroneously added in the previous diff. CPU was not supported yet and I haven't added anything for XPU. This comment itself is supposed to designate what is supported but I can also add it in the downstream APIs as well.